README.md

```diff
@@ -168,6 +170,35 @@ produces `terminal.png`:
 <!-- generated by script/regen-screenshots.sh -->
 ![play-msgcat-font-example](doc/images/screenshot-msgcat-font.png)
 
+## Toolkits
```

Comment:
Worth mentioning somewhere in here what toolkit supports which OSes?

Reply:
Currently, both java2d and skia seem to support the same OS's that membrane.term says there is support for:
> Membrane.term works on macOS and Linux operating systems.